scripts/build/dev-build.ps1

```diff
@@ -100,7 +103,7 @@ try {
             $skipTests = ""
         }
         Invoke-InLocation ".." {
-            Exec { & mvn clean install -P local-analyzer -D analyzer.configuration=$buildConfiguration $skipTests }
+            Exec { & mvn clean install --batch-mode -P local-analyzer -D analyzer.configuration=$buildConfiguration $skipTests }
```

Comment:
I compared the logs before & after and I don't see any change when running in `-buildJava` mode.
What's the difference? 
If it doesn't work as expected, let's remove it.

Reply:
the difference is when it downloads JARs from the cache  (the "Downloading..." message)- when they are already in the cache, you won't see it.

anyhow, I can remove it as locally we usually have cached assemblies